### ISSUE TITLE
Is C910 vulnerable to Spectre v2?

### ISSUE THREAD

**Issue by BigBrotherJu** [cispa/Security-RISC#2]
Looks like the `spectre` folder only demonstrates Spectre v1 (Exploiting Conditional Branches).

**Comment by s8lvg** [comment]
Yes the C910 is vulnerable to Spectre v2 given the predictor is enabled. In closing this issue once our research on this goes public and linking the code for Spectre v2 here.
